src/DIRAC/FrameworkSystem/Client/BundleDeliveryClient.py

```diff
@@ -115,15 +121,15 @@ def syncDir(self, bundleID, dirToSyncTo):
                 except OSError as e:
                     self.log.error("Could not sync dir:", str(e))
                     if dirCreated:
-                        self.log.info("Removing dir %s" % dirToSyncTo)
+                        self.log.info("Removing dir", dirToSyncTo)
                         os.unlink(dirToSyncTo)
                     buff.close()
-                    return S_ERROR("Certificates directory update failed: %s" % str(e))
+                    return S_ERROR(f"Certificates directory update failed: {e}")
 
         buff.close()
         self.__setHash(bundleID, dirToSyncTo, newHash)
         self.log.info("Dir has been synchronized")
-        return S_OK(True)
+        return S_OK()
```

Comment:
There is a `return S_OK(False)` above, and the docstring promises bools in the Value. If the Value value is not used anywhere, the docstring should be adapted and the False above removed as well.